server/src/packages/pas-form/pas-form.controller.ts

```diff
@@ -111,6 +111,9 @@ export const PAS_FORM_ATTRS = [
   'dcp_projectdescriptionproposedarea',
   'dcp_projectdescriptionsurroundingarea',
   'dcp_projectattachmentsotherinformation',
+
+  // Internals
+  '_dcp_projectname_value',
```

Comment:
What's the difference between this and `dcp_projectname`?

Reply:
Because we need that data in the request.

`_dcp_projectname_value` is the project identifier.